pennylane_sf/expectations.py

```diff
@@ -36,8 +36,7 @@
 import numpy as np
 
 import strawberryfields as sf
-from strawberryfields.backends.states import BaseFockState
```

Comment:
This file includes changes akin to on the branch for `qml.probs`. Further such changes include the removal of tests raising an error for `TensorN`